### PR TITLE
Deploy New Task Definition

### PR DESCRIPTION
**Pull request by squidurs**
Updated deployment step to ensure ECS service uses the latest Docker image by updating the task definition and triggering a new deployment.

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -34,7 +35,26 @@ jobs:
           docker build -t $ECR_REPOSITORY:$IMAGE_TAG .
           docker tag $ECR_REPOSITORY:$IMAGE_TAG $ECR_REGISTRY/$ECR_REPOSITORY:$IMAGE_TAG
           docker push $ECR_REGISTRY/$ECR_REPOSITORY:$IMAGE_TAG
+          echo "::set-output name=image::$ECR_REGISTRY/$ECR_REPOSITORY:$IMAGE_TAG"  
 
-      - name: Update ECS service
+      - name: Download task definition
```
Comment (darekj28) [inline .github/workflows/main.yml:40]: How did you test this change? Or how are you sure did it work? For configuration changes like this, please attach a Loom in the PR to show that this change works.

**Review by darekj28 — CHANGES_REQUESTED** [review]

File: .github/workflows/main.yml (hunk 1 above)
Comment (squidurs) [reply .github/workflows/main.yml:40]: I replicated the workflow configuration and triggered it to run when I pushed to my working-branch. This allowed me to confirm that the changes work as expected. Attached is a screenshot showing the successful deployment for verification.
Screenshot 2024-11-13 214337

Comment (squidurs) [reply .github/workflows/main.yml:40]: Here's a loom video as well:
https://www.loom.com/share/27ea3050c7b3433e871f832c7af0f498?sid=cf58bd39-0567-4045-baff-3678cb99a76b

You can skip to 1:16 to see that it worked

**Comment by darekj28** [comment]
Looks good!

**Review by darekj28 — APPROVED** [review]

**Pull request merged**